grid-proxy/internal/explorer/db/postgres.go

```diff
@@ -934,3 +934,9 @@ func (p *PostgresDatabase) UpsertNodesGPU(ctx context.Context, nodesGPU []types.
 	}
 	return nil
 }
+
+func (p *PostgresDatabase) GetLastNodeTwinID(ctx context.Context) (uint32, error) {
```

Comment:
Maybe we should change it to be GetGreaterThanTwinID and use the twinID we checked last? this should get us the list of ids that we can directly send?

Reply:
Initially, we thought we only need the latest ID and all new nodes will be from last checked to the latest ID however this is not right as twin IDs are shared with other entities so node twin IDs don't have to be incremented one by one. So, this approach is probably better.